src/migrate/index.js

```diff
@@ -119,10 +166,22 @@ export default class Migrator {
       .then((batchNo) => {
         return this._waterfallBatch(batchNo, migrations, direction)
       })
+      .then(() => this._unlockMigrations())
```

Comment:
Add another error handler `.catch(LockingError, () => {})` which will do nothing in case if locking failed